pages/about.html

```diff
@@ -4,9 +4,59 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
 </head>
-<body>
-  
+<body style="background-color:lightcyan;">
```

Comment:
Avoid putting style directly in your HTML. This is best placed in your CSS file instead.